page/docs/guide/json-rpc-api.md

```diff
@@ -59,4 +59,8 @@ Please note however, that the `pending` block will be the same block as the `lat
     "block_hash": "0x1234" // Must be hash of the latest block
   }
 }
-```
\ No newline at end of file
+```
+
+## starknet_getEvents
+
+**Disclaimer!** JSON-RPC specifications are not completely in sync with those of gateway. While `starknet_getEvents` is supported for the pending block, the official schmea does not allow the block hash and the block number in the response to be empty or anything other than a number. Since these values are undefined for the pending block and since they must be set to something, we decided to go with the compromise of setting them to zero-values.
```

Comment:
schmea -> schema

Reply:
Addressed in https://github.com/Shard-Labs/starknet-devnet/pull/399/commits/9aef3332f64c6eb1c911203607339f73c7f7a022